src/apps/Map/components/SearchBar.js

```diff
@@ -40,6 +41,17 @@ const SearchInput = styled.input`
   border-radius: 2px;
   box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.24), 0 0 2px 0 rgba(0, 0, 0, 0.12);
   text-align: center;
+  position: absolute;
+`;
+
+const SearchIcon = styled.span`
+  background-image: url('${SearchIconImage}');
+  background-repeat: no-repeat;
+  width: 53px;
+  height: 53px;
+  position: absolute;
+  right: 0;
+  background-position: center center;
 `;
```

Comment:
Size: This should IMO be smaller, more in line with the Burger Icon.
Adding `background-size: 25px;` looks good to me. This does not have to be too prominent, IMO.

---

This needs a `pointer-events: none; ` (more at https://tailwindcss.com/docs/pointer-events) so that a click on the icon actually is a click on the input.

---

Can be `height: 100%;` which is one less absolute value.

Off topic about the width px: I am trying to only use em/rem values and px only in special cases. This one is OK IMO.

Reply:
Fixed those issues.